src/hotspot/share/runtime/java.cpp

```diff
@@ -72,6 +72,7 @@
 #include "runtime/timer.hpp"
 #include "runtime/vmOperations.hpp"
 #include "services/memTracker.hpp"
+#include "services/stathist.hpp"
```

Comment:
Should be prepended by a // SapMachine 2019-09-01: vitals. comment

Reply:
Thank you Christoph.